workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -135,113 +158,134 @@ public int Run()
                 WorkingDirectory = workingDirectory
             };
 
-            using (var process = Process.Start(info))
+            var process = Process.Start(info);
+
+            if (process == null)
             {
-                if (process == null)
+                throw new Exception(
+                    $"Failed to run {info.FileName} {info.Arguments}\nIs the .NET Core SDK installed?");
```

Comment:
do we always need the .Net Core SDK when running a process?

Reply:
This was here a while ago, so I don't have a clue :)
My guess is that we use this in a few places to run .NET related processes.

Reply:
Yeah this isn't a valid error message given this method

Reply:
Fair point, this function has changed significantly since this message was added, I'll generalise the message